buildenv

```diff
@@ -95,5 +206,6 @@ zopen_get_version()
 {
   # Modify to echo the version of your tool/library
   # Rather than hardcoding the version, obtain the version by running the tool/library
+#echo "${ZOPEN_DEV_TAG}" | cut -d'-' -f2
```

Comment:
Should this be uncommented?

Reply:
No, i have corrected the version fetching now.